modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -856,6 +858,7 @@ add-field-set=Add Field Set
 add-fields=Add Fields
 add-fields-to-be-reused-on-the-newly-created-entry=Add fields to be reused on the newly created entry.
 add-fields-to-show-in-your-view=Add fields to show in your view.
+add-fields-to-unique-composite-key=Add Fields to Unique Composite Key
```

Comment:
@pedro-oliveira446 is it `Unique Composite Key` or `Composite Unique Key`? We are using both in this PR.